pro/src/components/VenueForm/CollectiveVenueInformations/CollectiveDmsTimeline/__specs__/CollectiveDmsTimeline.spec.tsx

```diff
@@ -5,16 +5,54 @@ import { renderWithProviders } from 'utils/renderWithProviders'
 
 import CollectiveDmsTimeline from '..'
 
-const renderCollectiveDmsTimeline = () => {
-  renderWithProviders(<CollectiveDmsTimeline />)
+const renderCollectiveDmsTimeline = ({
+  bookingDmsStatus,
+}: {
+  bookingDmsStatus: string
+}) => {
+  renderWithProviders(
+    <CollectiveDmsTimeline bookingDmsStatus={bookingDmsStatus} />
+  )
 }
 
 describe('CollectiveDmsTimeline', () => {
   it('should render draft state', () => {
-    renderCollectiveDmsTimeline()
+    renderCollectiveDmsTimeline({ bookingDmsStatus: 'DRAFT' })
 
     expect(
       screen.getByText('Déposez votre demande de référencement')
     ).toBeInTheDocument()
   })
+
+  it('should render steps for submitted dms status', () => {
```

Comment:
Suggestion : 

On pourrait utiliser `it.each` pour factoriser les tests. Il y a un exemple dans `CollectiveOfferStatusCell.spec.tsx`